### PR TITLE
Fix the sendMessage function attachments issue

### PR DESCRIPTION
**Pull request by SanduDS**
# Description

Fixing the sendMessage function issue when attempting to send mail without attachments.
Fixes https://github.com/ballerina-platform/ballerina-extended-library/issues/361

One line release note: 
- Fixing sendMessage function attachement related issue.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
- All tests passed

**Test Configuration**:
* Ballerina Version: SwanLake 2021.0.2
* Operating System: Ubuntu 
* Java SDK: JDK 11

# Checklist:

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: outlookmail/Ballerina.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 distribution = "2201.0.0"
```
Comment (sachinira) [inline outlookmail/Ballerina.toml:2]: Check Ballerina version

File: outlookmail/tests/test.bal [hunk 2]
```diff
@@ -234,9 +234,7 @@ function testSendMessage() {
     var output = oneDriveClient->sendMessage(messageReq);
```
Comment (sachinira) [inline outlookmail/tests/test.bal:234]: oneDriveClient -> outlookClient :relaxed:

File: outlookmail/utils.bal [hunk 3]
```diff
@@ -113,30 +113,32 @@ isolated function addOdataFileType(MessageContent|FileAttachment messageContent)
...
+        foreach int i in 0 ... attachments.length() {
+            FileAttachment attachment = attachments.remove(0);
+            FileAttachment attachmentTemp = {
+                contentBytes: attachment.contentBytes,
+                name: attachment.name,
+                contentType: attachment.contentType,
+                "@odata.type": "#microsoft.graph.fileAttachment"
+            };
+            if (attachment?.id is string) {
+                attachmentTemp.id = attachment?.id.toString();
```
Comment (kapilraajP) [inline outlookmail/utils.bal:126]: Can we use `ensureType` instead of `toString`?

File: outlookmail/utils.bal [hunk 4]
```diff
@@ -113,30 +113,32 @@ isolated function addOdataFileType(MessageContent|FileAttachment messageContent)
...
-            attachmentTemp.isInline = <boolean>attachment?.isInline;
-        }
-        if (attachment?.lastModifiedDateTime is string) {
-            attachmentTemp.lastModifiedDateTime = attachment?.lastModifiedDateTime.toString();
-        }
-        if (attachment?.size is int) {
-            attachmentTemp.size = <int>attachment?.size;
+    if (attachments.length() > 0) {
+        foreach int i in 0 ... attachments.length() {
+            FileAttachment attachment = attachments.remove(0);
```
Comment (abeykoon) [inline outlookmail/utils.bal:118]: We are not getting the i th element, rather we always get 0th. 
Any reason for that?

**Review by abeykoon — APPROVED** [review]

**Pull request merged**